### PR TITLE
Update 2021-04-17-traefik-portainer-ssl.md

### PR DESCRIPTION
**Pull request by eXsoR65**
Added a note about replacing USER with your username and PASSWORD with your password for the Basic Auth command and changed the wording for were to place it in the docker-compose file.

### CODE REVIEW

File: _posts/2021-04-17-traefik-portainer-ssl.md [hunk 1]
```diff
@@ -73,11 +73,13 @@ sudo apt install apache2-utils
 ```
 
 ```bash
-echo $(htpasswd -nb USER PASSWORD) | sed -e s/\\$/\\$\\$/g
+echo $(htpasswd -nb <USER> <PASSWORD>) | sed -e s/\\$/\\$\\$/g
 ```
+NOTE: Replace `<USER>` with your username and `<PASSWORD>` with your password to be hashed.
 
-use this in your `docker-compose.yml` (`USER:BASIC_AUTH_PASSWORD`)
+Paste the out put in your `docker-compose.yml` in line (`traefik.http.middlewares.traefik-auth.basicauth.users=<USER>:<HASHED-PASSWORD>`)
```
Comment (timothystewart6) [inline _posts/2021-04-17-traefik-portainer-ssl.md:80]: should be `output` :)

**Comment by timothystewart6** [comment]
Thanks for this!  One small comment other than that good to go.  Thank you!

File: _posts/2021-04-17-traefik-portainer-ssl.md (hunk 1 above)
Comment (eXsoR65) [reply _posts/2021-04-17-traefik-portainer-ssl.md:80]: ah yes..

**Comment by timothystewart6** [comment]
Thanks @eXsoR65 !

**Pull request merged**